distributed/protocol/torch.py

```diff
@@ -7,7 +7,12 @@
 @dask_serialize.register(torch.Tensor)
 def serialize_torch_Tensor(t):
     requires_grad_ = t.requires_grad
-    header, frames = serialize(t.detach_().numpy())
+
+    try:
+        header, frames = serialize(t.detach_().numpy())
+    except RuntimeError:
+        header, frames = serialize(t.numpy())
```

Comment:
Why not an use an if-statement instead? It's known when the error will be thrown: when `t.requires_grad` is true.

Reply:
I changed this block to use an if-statement.